app/assets/javascripts/angular/controllers/mobile/espionage.js

```diff
@@ -0,0 +1,152 @@
+(function() {
+  angular.module('wtsApp').controller('espionageCtrl', ['$scope', '$http', '$interval', function($scope, $http, $interval) {
```

Comment:
This is mostly the same as Messages so at some point in the future I'll make them a service or refactor it so both can use the same thing. Since the differences are basically "Espionage can't send messages". It would just be including functions that we don't / can't actually use, which is pretty no harm no foul.